### PR TITLE
Include PDB CRYST1 information in the header.

### PR DESCRIPTION
**Pull request by mok0**
I submitted a pull request a long time ago. I was asked to write a unit test, which I did. I don't know what happened to the pull request, I can't find it. 

The change was discussed on on the biopython-dev list 9-11. August 2013, and people seemed to ok it.

The branch is rebased off the most recent master.

### CODE REVIEW

**Comment by peterjc** [comment]
You closed the old pull request https://github.com/biopython/biopython/pull/291 two hours ago (probably by mistake).

We had a brief discussion with @JoaoRodrigues about default values, but I was hoping for him and/or @etal to review the pull request.

@mok0  two further questions for you (assuming Joao and Eric are happy):

- Are you happy to be named in the ``NEWS`` and ``CONTRIB`` files?
- Are you happy to dual license your contributions under both the current Biopython License Agreement AND the 3-clause BSD license (see #898).

Thanks!

**Comment by mok0** [comment]
@peterjc, thanks for your quick reply! To answer your two questions, "yes" to both :-) 

I deleted the branch "cryst1" because it had an unclean history, it contained a bunch of upstream merges. Apparently that also closed the old merge request. Very smart, GitHub, I hadn't thought about that.

I see now that there are some Travis issues I need to deal with, it's unclear to me ATM what it's about.

**Comment by peterjc** [comment]
The first thing TravisCI picked up is Python coding style - we now enforce some of the PEP8 guidelines automatically:

```
 pep8 --ignore E122,E123,E126,E127,E128,E129,E501,E731 Bio/
Bio/PDB/parse_pdb_header.py:145:15: E203 whitespace before ':'
Bio/PDB/parse_pdb_header.py:146:17: E203 whitespace before ':'
Bio/PDB/parse_pdb_header.py:147:21: E203 whitespace before ':'
Bio/PDB/parse_pdb_header.py:253:1: E112 expected an indented block
Bio/PDB/parse_pdb_header.py:253:3: E227 missing whitespace around bitwise or shift operator
Bio/PDB/parse_pdb_header.py:253:7: E225 missing whitespace around operator
Bio/PDB/parse_pdb_header.py:254:21: E116 unexpected indentation (comment)
Bio/PDB/parse_pdb_header.py:255:21: E113 unexpected indentation
Bio/PDB/parse_pdb_header.py:256:3: E225 missing whitespace around operator
Bio/PDB/parse_pdb_header.py:256:5: E225 missing whitespace around operator
Bio/PDB/parse_pdb_header.py:256:7: E225 missing whitespace around operator
Bio/PDB/parse_pdb_header.py:257:21: E116 unexpected indentation (comment)
Bio/PDB/parse_pdb_header.py:257:21: E265 block comment should start with '# '
Bio/PDB/parse_pdb_header.py:258:21: E113 unexpected indentation
Bio/PDB/parse_pdb_header.py:258:39: E225 missing whitespace around operator
Bio/PDB/parse_pdb_header.py:259:9: E901 IndentationError: unindent does not match any outer indentation level
ERROR: InvocationError: '/home/travis/build/biopython/biopython/.tox/pep8/bin/pep8 --ignore E122,E123,E126,E127,E128,E129,E501,E731 Bio/'
```

You can do ``pip install pep8`` in order to run this check locally. Also ``pip install autopep8`` gives you the ``autopep8`` tool which can be handy for fixing some of these issues semi-automatically.

File: Bio/PDB/parse_pdb_header.py [hunk 1]
```diff
@@ -246,8 +250,23 @@ def _parse_pdb_header_list(header):
                 try:
                     dict['resolution'] = float(r)
                 except:
+<<<<<<< HEAD
```
Comment (peterjc) [inline Bio/PDB/parse_pdb_header.py]: It looks like you committed a merge without resolving it - you'll need to clean up this in order for the Python code to work.

**Comment by mok0** [comment]
How embarrassing 😖... didn't notice the unresolved conflict. I fixed that and the PEP8 issues. I also ran the test suite, it failed on 2 unrelated modules, I think due to missing Entrez tools on my laptop. I've pushed a new version of `parse_pdb_header.py`.

**Pull request closed without merging**